doc/sphinxext/gallery_order.py

```diff
@@ -6,33 +6,48 @@
 from sphinx_gallery.sorting import ExplicitOrder
 
 # Gallery sections shall be displayed in the following order.
-# Non-matching sections are inserted at UNSORTED
-explicit_order_folders = [
-                          '../examples/lines_bars_and_markers',
-                          '../examples/images_contours_and_fields',
-                          '../examples/subplots_axes_and_figures',
-                          '../examples/statistics',
-                          '../examples/pie_and_polar_charts',
-                          '../examples/text_labels_and_annotations',
-                          '../examples/pyplots',
-                          '../examples/color',
-                          '../examples/shapes_and_collections',
-                          '../examples/style_sheets',
-                          '../examples/axes_grid1',
-                          '../examples/axisartist',
-                          '../examples/showcase',
-                          '../tutorials/introductory',
-                          '../tutorials/intermediate',
-                          '../tutorials/advanced',
-                          '../plot_types/basic',
-                          '../plot_types/arrays',
-                          '../plot_types/stats',
-                          '../plot_types/unstructured',
-                          '../plot_types/3D',
-                          'UNSORTED',
-                          '../examples/userdemo',
-                          '../tutorials/provisional',
-                          ]
+# Non-matching sections are inserted at the unsorted position
+
+UNSORTED = "unsorted"
+
+examples_order = ['../examples/lines_bars_and_markers',
+                  '../examples/images_contours_and_fields',
+                  '../examples/subplots_axes_and_figures',
+                  '../examples/statistics',
+                  '../examples/pie_and_polar_charts',
+                  '../examples/text_labels_and_annotations',
+                  '../examples/pyplots',
+                  '../examples/color',
+                  '../examples/shapes_and_collections',
+                  '../examples/style_sheets',
+                  '../examples/axes_grid1',
+                  '../examples/axisartist',
+                  '../examples/showcase',
+                  UNSORTED,
+                  '../examples/userdemo']
```

Comment:
Optional: I like this long-list formatting better because it creates less ragged code (no very long leading whitespace, no differences in indentation because of different-length list names)

```suggestion
examples_order = [
    '../examples/lines_bars_and_markers',
    '../examples/images_contours_and_fields',
    '../examples/subplots_axes_and_figures',
    '../examples/statistics',
    '../examples/pie_and_polar_charts',
    '../examples/text_labels_and_annotations',
    '../examples/pyplots',
    '../examples/color',
    '../examples/shapes_and_collections',
    '../examples/style_sheets',
    '../examples/axes_grid1',
    '../examples/axisartist',
    '../examples/showcase',
    UNSORTED,
    '../examples/userdemo',
]
```